app/components/system-status/component.js

```diff
@@ -1,38 +1,48 @@
-import Component from '@ember/component';
-import { task } from 'ember-concurrency';
-import ENV from 'irene/config/environment';
-import {isNotFoundError} from 'ember-ajax/errors';
+import Component from "@ember/component";
+import { task } from "ember-concurrency";
+import ENV from "irene/config/environment";
+import { isNotFoundError } from "ember-ajax/errors";
 
 export default Component.extend({
-
   isStorageWorking: false,
   isDeviceFarmWorking: false,
+  isAPIHostWorking: false,
 
   localClassNameBindings: [
-    'isStorageWorking:storage-operational',
-    'isDeviceFarmWorking:devicefarm-operational'],
+    "isStorageWorking:storage-operational",
+    "isDeviceFarmWorking:devicefarm-operational",
+  ],
```

Comment:
This is unused, right?

Reply:
Yes. Have removed this in the latest commit.